csrc/transform_replay.cpp

```diff
@@ -519,29 +519,6 @@ std::pair<TensorDomain*, size_t> TransformReplay::replayPasC(
     }
   }
 
-  if (producer->hasAllocation()) {
```

Comment:
This check is removed. If the replay makes the allocation domain no longer valid, then the ctor of `TensorDomain` will detect that.